src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -45,8 +43,6 @@ public void assertIfAllRecoveredSlotHasCorrectSizeAndBackground(List<WebElement>
         Assertion.assertTrue(ad.getCssValue("background").contains(expectedRecoveredLB), "TOP_LEADERBOARD is not correctly recovered!");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "background" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)